Library/Homebrew/extend/os/linux/dev-cmd/update-test.rb

```diff
@@ -6,6 +6,10 @@ module Homebrew
 
   module_function
 
+  class << self
+    alias generic_git_tags git_tags
+  end
```

Comment:
This works but is really ugly and hacky. Do we really want to do this or is there a better way to handle `extend/os` stuff with `module_function`?

Reply:
@apainintheneck That's the best way to hand this with `module_function`. The alternative would be to wrap all methods in the `class << self` and drop `module_function` use entirely.